instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientCommonAttributesExtractor.java

```diff
@@ -41,7 +41,7 @@ public void onStart(AttributesBuilder attributes, Context parentContext, REQUEST
     }
     if (SemconvStability.emitOldDatabaseSemconv()) {
       internalSet(attributes, DB_USER, getter.getUser(request));
-      internalSet(attributes, DB_NAME, getter.getName(request));
+      internalSet(attributes, DB_NAME, getter.getNamespace(request));
```

Comment:
it's probably correct here to call the (deprecated) `getName`, since under `emitOldDatabaseSemconv`